sbncode/CAFAna/Core/CMakeLists.txt

```diff
@@ -3,8 +3,13 @@ include_directories( $ENV{IFDHC_FQ_DIR}/inc )
 # Allow "StandardRecord/" to find the right directory
 include_directories(..)
 
+include_directories($ENV{OSCLIB_INC})
+
 art_make(LIBRARY_NAME CAFAnaCore
          LIB_LIBRARIES StandardRecordProxy
+                       # TODO how am I supposed to link to an external library?
```

Comment:
@wesketchum just ${OSCLIB} didn't work here like I expected. I think there a couple other instances of this pattern too.